IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -174,7 +183,8 @@ - (MSIDToken *)getATForAccount:(MSIDAccount *)account
             }
             return nil;
         }
-        if (![v2Params.scopes isSubsetOfOrderedSet:token.scopes])
+        
+        if (![((MSIDAADV2RequestParameters *)parameters).scopes isSubsetOfOrderedSet:token.scopes])
```

Comment:
nit: can you declare MSIDAADV2RequestParameters separately?

Reply:
sure